opentech/public/people/templates/people/person_page.html

```diff
@@ -1,60 +1,61 @@
 {% extends "base.html" %}
-
+{% block page_title %}People{% endblock %}
```

Comment:
If this is the page title it should probably show the persons name so users can identify different tabs.

Reply:
The `<title>` for the page should still show the persons name as `{% block title %}` hasn't changed. See - https://otf-dev.torchboxapps.com/people/person/

Perhaps the naming of the blocks should be reviewed as we have `{% block title %}` and `{% block page_title %}`?